suite-native/alerts/src/alertsAtoms.ts

```diff
@@ -6,8 +6,16 @@ import { ButtonAccessory, ButtonColorScheme, PictogramVariant } from '@suite-nat
 import { IconName } from '@suite-native/icons';
 import { NativeSpacing } from '@trezor/theme';
 
+export type AlertType =
+    | 'autoEject'
+    | 'bluetoothPairing'
+    | 'bluetoothPermissions'
+    | 'connectDevice'
+    | 'detectDeviceError';
```

Comment:
This might be more exact.
```suggestion
    | 'deviceError';
```

Reply:
https://github.com/trezor/trezor-suite/commit/9d78fd8d18ba087bdf3a63a3ff2a7094dfb27a09